src/game/Strategic/Map_Screen_Interface_Map.cc

```diff
@@ -139,14 +139,14 @@ INT32 iZoomY = 0;
 #define MAP_Y_ICON_OFFSET   1
 
 // Arrow Offsets
-#define UP_X                13
-#define UP_Y                 7
-#define DOWN_X               0
-#define DOWN_Y              -2
-#define RIGHT_X             -2
-#define RIGHT_Y             11
-#define LEFT_X               2
-#define LEFT_Y               5
+//#define UP_X                13
+//#define UP_Y                 7
+//#define DOWN_X               0
+//#define DOWN_Y              -2
+//#define RIGHT_X             -2
+//#define RIGHT_Y             11
+//#define LEFT_X               2
+//#define LEFT_Y               5
```

Comment:
Why not just delete them?

Reply:
I kept all obsolete coordinate defines commented out like this... in case something somewhere uses hardcoded 11 or -2 (because there are a lot of places that do that when calculating coordinates) so it's easier to find it. Once I'm done with all the changes will go through commits and remove ugly stuff like this.